setup.py

```diff
@@ -18,10 +18,11 @@
     packages=find_packages(where='src'),
     package_dir={'': 'src'},
     install_requires=[
-        'py4j',
         'numpy',
         'pandas',
+        'py4j',
         'pyarrow>=0.8.0',
+        'pyspark',
```

Comment:
Is this safe / why did we avoid this before? (Were there concerns about pip's eager upgrade behavior?)

Reply:
I don't recall a reason why this wasn't included. This project does require it to work.